bin/cml/repo/prepare.js

```diff
@@ -0,0 +1,31 @@
+const kebabcaseKeys = require('kebabcase-keys');
```

Comment:
delete this file?

Reply:
https://github.com/iterative/cml/pull/1073#discussion_r957920154